collections/_article/dev-snapshot-godot-4-3-beta-1.md

```diff
@@ -0,0 +1,255 @@
+* Core: Add typed array support for binary serialization ([GH-78219](https://github.com/godotengine/godot/pull/78219)).
+* Use black for font outlines by default instead of white ([GH-54641](https://github.com/godotengine/godot/pull/54641)).
+* Change TileMapEditor to TileMapLayerEditor ([GH-87379](https://github.com/godotengine/godot/pull/87379)).
+* Remove `bone_pose_updated` signal and replace it with the `skeleton_updated` signal ([GH-90575](https://github.com/godotengine/godot/pull/90575)).
+* Implement a base class SkeletonModifier3D as refactoring for nodes that may modify Skeleton3D ([GH-87888](https://github.com/godotengine/godot/pull/87888)).
+* Rework AnimationNode process for retrieving the semantic time info ([GH-87171](https://github.com/godotengine/godot/pull/87171)).
+* Add `AnimationMixer::capture()` and `AnimationPlayer::play_with_capture()` as substitute of update mode capture ([GH-86715](https://github.com/godotengine/godot/pull/86715)).
+* Fix TrackCache conflict when tracks have same name but different type ([GH-86687](https://github.com/godotengine/godot/pull/86687)).
+* Reverse Z: Some shaders will need to change to accommodate the new depth buffer format. More details [here](https://godotengine.org/article/introducing-reverse-z/).
+* Rework the auto translation system ([GH-87530](https://github.com/godotengine/godot/pull/87530)).
+* C#: Implement InvariantCulture on Variant strings ([GH-89547](https://github.com/godotengine/godot/pull/89547)).
+* Freed objects are now different than null in comparison operators ([GH-73896](https://github.com/godotengine/godot/pull/73896)).
+* Unbind GDExtension methods that can't reasonably be used ([GH-88418](https://github.com/godotengine/godot/pull/88418)).
+* Fix some AcceptDialog argument types. Previous erroneous calls will now error instead of do nothing ([GH-89419](https://github.com/godotengine/godot/pull/89419)).
+
+### 2D
+
+#### Huge improvement to pixel stability for pixel art games
+After much community discussion and the hard work of several contributors, we have merged a [PR](https://github.com/godotengine/godot/pull/87297) that we think resolves many of the outstanding issues with pixel stability when making pixel art games. As before, it relies on using the `rendering/2d/snap/snap_2d_transforms_to_pixel` project setting. If you are making a pixel art game, please test this new release carefully and let us know how it goes.
+
+#### 2D physics interpolation
+Fixed timestep a.k.a. physics interpolation is now implemented for 2D ([GH-88424](https://github.com/godotengine/godot/pull/88424)), forward-ported from the version merged for Godot 3.6 last year ([GH-76252](https://github.com/godotengine/godot/pull/76252)).
+
+This will help address cases of position/camera jitter in 2D games, and should complement some of the pixel-art focused changes made in the 4.3 dev 4 snapshot.
```

Comment:
Should there be a word on 3D physics interpolation already being worked on for a future release? This is bound to be a common question after all.

Reply:
```suggestion
Fixed timestep a.k.a. physics interpolation is now implemented for 2D ([GH-88424](https://github.com/godotengine/godot/pull/88424)), forward-ported from the version merged for Godot 3.6 last year ([GH-76252](https://github.com/godotengine/godot/pull/76252)).

This will help address cases of position/camera jitter in 2D games, and should complement some of the pixel-art focused changes made in the 4.3 dev 4 snapshot.

3D physics interpolation is also in the works for a future Godot release ([GH-92391](https://github.com/godotengine/godot/pull/92391), possibly 4.4.
```